### PR TITLE
Fix removal of yes no answers

### PR DESCRIPTION
**Pull request by brandenchan**
Previously, yes no answers in the Natural Questions dataset were being converted into span answers (#529). This PR fixes this problem.

### CODE REVIEW

File: farm/data_handler/processor.py [hunk 1]
```diff
@@ -1744,10 +1745,13 @@ def _apply_tokenization(dictionary, tokenizer):
             external_id = question["id"]
             question_text = question["question"]
             for answer in question["answers"]:
-                if answer["text"] == "":
-                    answer_type = "no_answer"
+                if 'answer_type' in answer.keys() and answer['answer_type'] in ['yes', 'no', 'span', 'no_answer']:
```
Comment (Timoeller) [inline farm/data_handler/processor.py:1748]: I am unhappy about having the answer types hard coded here as a list. We should somehow link the already defined answer_type_list attribute of NaturalQuestionsProcessor.

**Review by Timoeller — CHANGES_REQUESTED** [review]
Lets remove hard coded answer types as suggested in comment.

File: farm/data_handler/processor.py (hunk 1 above)
Comment (tholor) [reply farm/data_handler/processor.py:1748]: Can we finish this before the next release?

**Comment by Timoeller** [comment]
Hey sorry @brandenchan but this is not the best solution. We have an empty list as default arg which is an anti pattern, see https://docs.python-guide.org/writing/gotchas/

Otherwise the solution is ok with me.

**Pull request merged**